backend/api/src/db/category.rs

```diff
@@ -30,6 +33,15 @@ order by index
         children: vec![],
         image_count: it.image_count as u64,
         jig_count: it.jig_count as u64,
+        user_scopes: {
+            let scopes = it
+                .user_scopes
+                .into_iter()
+                .map(|x| UserScope::try_from(x).unwrap())
+                .collect();
+
+            scopes
+        },
```

Comment:
style comment: this and lines 77-82 do the same thing. Would prefer using the latter for both as it avoids creating a new scope and declaring a short lived variable